demos/common/python/openvino/model_zoo/model_api/models/model.py

```diff
@@ -38,6 +46,62 @@ def __init__(self, model_adapter):
         self.model_adapter = model_adapter
         self.inputs = self.model_adapter.get_input_layers()
         self.outputs = self.model_adapter.get_output_layers()
+        for name, parameter in self.parameters().items():
+            self.__setattr__(name, parameter.default_value)
+        self.load_config(configuration if configuration else {})
+        self.model_loaded = False
+        if preload:
```

Comment:
I suggest to make the `preload` as `OpenvinoAdapter` argument. The default value is `True`, so the model is loaded to the device in the adapter constructor. The `False` value will be set for models with reshape, and for such model wrappers the loading is called after the reshape.

Also, the passing of `preload` to each constructor is might be discouraged, for instance, the OVMS does not load the model to the device at all, so the behavior of model wrapper is undefined for this case. 

Plus, delete the `load()` call in `AsyncPipeline`, please